apps/src/lib/kits/maker/Button.js

```diff
@@ -19,4 +19,25 @@ export default function Button(opts) {
     get: () => this.value === 1
   });
 }
-Button.inherits(five.Button);
+PlaygroundButton.inherits(five.Button);
+
+export function MicroBitButton(board) {
+  // There are six button events, ['', 'down', 'up', 'click', 'long-click', 'hold']
```

Comment:
What does `''` mean as a button event?

Reply:
Unknown. I referenced this from the MicroBit firmata tests -> https://github.com/microbit-foundation/microbit-firmata/blob/fd572115cf69a820d0d814736f8572243f19af7a/client/mbTests.js#L325
and having ' ' makes the rest of the events align.